lavamoat/browserify/beta/policy.json

```diff
@@ -1839,16 +1839,6 @@
         "safe-event-emitter": true
       }
     },
-    "eth-eip712-util-browser": {
```

Comment:
Any reason why these policy changes are in here?

Reply:
When I ran the extension, it was showing a white screen. When I opened the Console tab from background.html, there was a JS error. After running `yarn lavamoat:auto` and `yarn start` it all works well. That's why I committed the changes, so people don't need run `yarn lavamoat:auto` and wait for a long time until it's done.

If there is a different way to handle this, let me please know.

Reply:
Update: maybe I had that LavaMoat issue after I switched from another local branch. I've removed the policies from this PR and it seems that locally it's loading correctly now.